### PR TITLE
Added dev and devwatch scripts to easy the webapp+redux development

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "form-data": "2.3.1",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
+    "jquery": "3.2.1",
```
Comment (enahum) [inline package.json]: Why would you need jquery here? can we please remove it

File: package.json [hunk 2]
```diff
@@ -43,6 +44,8 @@
   },
   "scripts": {
     "build": "babel src --out-dir .",
+    "dev": "babel src --out-dir ../mattermost-webapp/node_modules/mattermost-redux",
+    "dev:watch": "babel --watch src --out-dir ../mattermost-webapp/node_modules/mattermost-redux",
```
Comment (jwilander) [inline package.json]: Is it possible to have the location of the webapp repo configurable via an environment variable?

File: package.json (hunk 1 above)
Comment (jespino) [reply package.json]: Sorry, fixed, was part of an afternoon of a lot of tries with different approaches.

**Comment by jespino** [comment]
@enahum added the README paragraph about compiling.

**Pull request merged**